answer_files/2019_core/Autounattend.xml

```diff
@@ -50,8 +50,8 @@
                 <OSImage>
                     <InstallFrom>
                         <MetaData wcm:action="add">
-                            <Key>/IMAGE/NAME</Key>
-                            <Value>Windows Server 2019 SERVERDATACENTERCORE</Value>
+                            <Key>/IMAGE/INDEX</Key>
+                            <Value>1</Value>

```

Comment:
Awesome. It worked with it. I also think this is a more stable way to select the image instead of the name.

Reply:
I'll try this with the Eval ISO if it still works.

Reply:
The /IMAGE/INDEX 1 also works with Eval ISO 👍